### PR TITLE
Add an option to force loopUnrollCount.

### PR DESCRIPTION
**Pull request by yjaelex**
Add an option to force loopUnrollCount.

### CODE REVIEW

File: tool/vfx/vfxSection.h [hunk 1]
```diff
@@ -1154,6 +1154,9 @@ class SectionShaderOption : public Section
         INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, vgprLimit, MemberTypeInt, false);
         INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, sgprLimit, MemberTypeInt, false);
         INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, maxThreadGroupsPerComputeUnit, MemberTypeInt, false);
+    #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 24
+        INIT_STATE_MEMBER_NAME_TO_ADDR(SectionShaderOption, forceLoopUnrollCount, MemberTypeInt, false);
+    #endif
 
         VFX_ASSERT(pTableItem - &m_addrTable[0] <= MemberCount);
```
Comment (linqun) [inline tool/vfx/vfxSection.h]: please increase the member count

**Comment by amdrexu** [comment]
Please fix issues and update the commit.

File: tool/vfx/vfxSection.h (hunk 1 above)
Comment (yjaelex) [reply tool/vfx/vfxSection.h]: Done!

**Comment by amdrexu** [comment]
Please click "Resolve Conversation" button in the future. Thus, we can skip check and know you have corrected the issue :)

**Comment by amdvlk-admin** [comment]
### Test summary for commit bde55552ef8704b68194791847c44ebc8f89c693
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 59095d3469e4fb7506da0e3d0cd78904ed71682f</li>
<li>XGL: dev 292ea8da8a985a72b9efe4db424f664b2e79d64d</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 836/836 (100.0%)</li>
<li>Failed: 0/836 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.1.2-cherry-picks)</summary><ul>


<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 312622/312622 (100.0%)</li>
<li>Failed: 0/312622 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 312622/312622 (100.0%)</li>
<li>Failed: 0/312622 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8f4aeaac01cd18babfb94ff8dd641435f2ce1f5d
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 1560d287f779b342e0019499dda85890cb07affc</li>
<li>XGL: dev b5440b6a385dbcb11739f1bdb89a94bbe5cd499f</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 902/902 (100.0%)</li>
<li>Failed: 0/902 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.1.2-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>No Results</li>

</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 8b26d7db2fc0c409250b32b71adf4012c8e8167a
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 1560d287f779b342e0019499dda85890cb07affc</li>
<li>XGL: dev b5440b6a385dbcb11739f1bdb89a94bbe5cd499f</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 902/902 (100.0%)</li>
<li>Failed: 0/902 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.1.2-cherry-picks)</summary><ul>


<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 312622/312622 (100.0%)</li>
<li>Failed: 0/312622 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**